sklearn/cluster/_affinity_propagation.py

```diff
@@ -449,7 +466,6 @@ def __init__(
         verbose=False,
         random_state=None,
     ):
-
```

Comment:
We usually don't make these types of changes if they are not triggered for PEP8 reasons.

Could you revert this part?

Reply:
Sorry, done.